yarn.lock

```diff
@@ -4317,6 +4317,16 @@
     "@typescript-eslint/typescript-estree" "4.33.0"
     debug "^4.3.1"
 
+"@typescript-eslint/parser@^5.43.0":
+  version "5.43.0"
+  resolved "https://packages.atlassian.com/api/npm/npm-remote/@typescript-eslint/parser/-/parser-5.43.0.tgz#9c86581234b88f2ba406f0b99a274a91c11630fd"
```

Comment:
need to change npm registry, it is using atlassian

Reply:
ah, good catch!